extensions/dashlane-vault/src/master-password.tsx

```diff
@@ -0,0 +1,66 @@
+import { Action, ActionPanel, Form, LocalStorage, Toast, showToast } from "@raycast/api";
+import { useEffect, useState } from "react";
+import { getMasterPassword } from "./helper/master-password";
+import { setKeychainItem } from "./lib/authcli";
+import { encryptVault } from "./lib/dcli";
+
+export default function SetMasterPassword() {
+  const [password, setPasswordInput] = useState("");
+  const [keyName, setKeyName] = useState("");
+  const [passwordExists, setPasswordExists] = useState(false);
+
+  const fetchKeyNameAndCheckPassword = async () => {
```

Comment:
In this command the user wants to set his master password. Why do we need to fetch and insert the current password?

Reply:
It's mainly for UI purposes.